plugin/trino-hive/src/test/java/io/trino/plugin/hive/coercions/TestDateCoercer.java

```diff
@@ -74,19 +77,39 @@ public void testThrowsExceptionWhenDateIsTooOld()
                 .hasMessageMatching(".*Coercion on historical dates is not supported.*");
     }
 
+    @Test
+    public void testDateToVarchar()
+    {
+        assertDateToVarcharCoercion(createUnboundedVarcharType(), LocalDate.of(2023, 1, 10), "2023-01-10");
+        assertDateToVarcharCoercion(createUnboundedVarcharType(), LocalDate.of(1899, 12, 31), "1899-12-31");
```

Comment:
just a thought, for opposite way conversion (VARCHAT => DATE) we only allow dates greater than 01.01.1900 is it ok to allow broader range here?